modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -4,21 +4,15 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: runtime, gateway, gatekeeper, security, policy, policies
 
-When you deploy an application, there could be a lapse during which the tracked resources exposed by the application are accessible, but the policies specified for those tracked resources are not yet applied, so unwelcome traffic may reach the them. GateKeeper prevents this situation by blocking the tracked resource until all policies have been retrieved and applied without errors. During this period, the API returns a 503 HTTP status code.
+When you deploy an application, the tracked resources exposed by the application might become accessible for a short amount of time due to a lapse in security. Because the policies specified for those tracked resources are not yet applied, the API might receive unwanted traffic. GateKeeper prevents this situation by blocking the tracked resource until all policies are retrieved and applied without errors. 
 
 == How Gatekeeper Works
 
-The Gatekeeper mechanism only applies when (re)starting the runtime or (re)deploying a tracked resource (ie one linked with an API instance through Autodiscovery).
+The Gatekeeper mechanism only applies when you (re)start the runtime or (re)deploy a tracked resource that is linked with an API instance through Autodiscovery. When Gatekeeper is engaged, each tracked resource is blocked ("503 - Service Unavailable" HTTP status code) until all API instance policies  are applied in API Manager without errors.
 
-When engaged, each tracked resource is blocked and a 503 - Service Unavailable HTTP status code is returned, until all policies that appear as applied in the API instance of API Manager are downloaded and applied without errors to the tracked resource.
+Resources without an associated Autodiscovery are ignored by Gatekeeper. Additionally, you cannot apply any policies to these resources and no analytics information is generated. If you loose connectivity after the Mule instance is (re)started or the tracked Mule application is (re)deployed, you use cached policies, configuration, and contracts even if Gatekeeper is not enabled. 
```

Comment:
```suggestion
Resources not linked to an API instance through Autodiscovery are ignored by Gatekeeper. Additionally, you cannot apply any policies to these resources, and no analytics information is generated. If you lose connectivity after the Mule instance is restarted or the tracked Mule application is redeployed, you use cached policies, configuration, and contracts, even if Gatekeeper is not enabled. 
```
Some of the rewording edits are an attempt to use just one wording rather than different wordings for the same thing, for example, Gatekeeper being "engaged" and "resources linked to an API instance through Autodiscovery".

As a reader, I don't know what you are implying by using "(re)"? I suggest that you choose the "re" version if that is most likely, or spell out the whole thing so that meaning is crystal clear: started or restarted, deployed or redeployed....

Reply:
Done! Thanks!